system/ee/ExpressionEngine/Controller/Files/AbstractFiles.php

```diff
@@ -227,13 +227,7 @@ protected function buildTable($files, $limit, $offset)
                 ),
             );
 
-            if (! ee('Permission')->can('edit_files')) {
-                unset($toolbar['view']);
```

Comment:
Didn't see this unset get reimplemented... does it need to be? @intoeetive

Reply:
The `$toolbar` array (created just above this code) does not have `view` key